sonatype-ci-release/src/main/scala/org/typelevel/sbt/TypelevelSonatypeCiReleasePlugin.scala

```diff
@@ -57,9 +61,27 @@ object TypelevelSonatypeCiReleasePlugin extends AutoPlugin {
 
       tags ++ branches
     },
+    tlCiReleaseStepSummaryTableInfo := {
+      Map("**Release version**" -> (ThisBuild / version).value)
+    },
```

Comment:
A couple other ideas for things we can put here:
1. The resolver it was published to (e.g. sonatype s01 snapshots), and how to configure it in sbt.
2. If populated, the value of `ThisBuild / apiURL`, since that will link to the scaladocs. Could be handy.

Reply:
2 is an easy task, but I'm not sure how to get the correct resolver in 1: the `resolvers` key is a `Seq[Resolver]` (and I bet is used just to resolve the deps I need to build my project, not as a target to publish to), and possibly the `publishTo` is just a project setting, not a build one right?

Reply:
I'm not sure on how to proceed since:
1) `resolvers` is a `Seq[Resolver]` and it might not contain the resolver the project is using since it's used to define "user additional resolvers"

```scala
val resolvers = settingKey[Seq[Resolver]]("The user-defined additional resolvers for automatically managed dependencies.").withRank(BMinusTask)
```
There's `fullResolvers`

```scala
val fullResolvers = taskKey[Seq[Resolver]]("Combines the project resolver, default resolvers, and user-defined resolvers.").withRank(CTask)
```
but it's a task

2) `publishTo` might be the thing, but it's a task like `fullResolvers`, not a setting
3) `Resolver` is an abstract class that merely exposes a `name: String` attribute, so extracting the exact information to put in the instructions is not as trivial as desired.

Reply:
Something like this might work
```scala
  override def projectSettings: Seq[Setting[_]] = Seq(
    publish := {
      val table: Map[String, String] = tlCiReleaseStepSummaryTableInfo.value
      val maybeResolver: Option[Resolver] = (ThisBuild / publishTo).value

      val textToRender: String =
        if (tlCiAddResolverInfoToSummary.value) {
          maybeResolver.fold(renderSummaryTable(table)) { r =>
            val text: String = renderSummaryTable(table + ("Resolver" -> r.toString))
            s"${text}\nTo configure your build to use this published version set\n`resolvers += ${r.toString}`"
          }
        } else renderSummaryTable(table)

      GitHubActionsPlugin.appendtoStepSummary(textToRender)

      publish.value
    }
  )
```

but requires an additional `tlCiAddResolverInfoToSummary` setting, plus it doesn't really solve the problem 3. We could write an exhaustive pattern match for resolvers, but do we really want to? :D